modules/core/src/main/java/org/locationtech/jts/index/strtree/STRtree.java

```diff
@@ -373,5 +402,91 @@ public Object nearestNeighbour(Envelope env, Object item, ItemDistance itemDist)
           ((ItemBoundable) minPair.getBoundable(1)).getItem()
       };
   }
+  private Object[] nearestNeighbour(BoundablePair initBndPair, double maxDistance, int k) 
```

Comment:
I guess not. Although @bjornharrtell is going to complain I think!  (because JSTS has problems with overloading...)